docs/canvas/canvas-elements.asciidoc

```diff
@@ -0,0 +1,259 @@
+[role="xpack"]
+[[element-intro]]
+== Showcase your data with elements
+
+Canvas _elements_ are the building blocks of your workpad. With elements, you can combine images, text, and visualizations to tell a story about your data.
+
+When you add elements to your workpad, you can:
+
+* <<connect-element-data,Connect the elements to your live data>>
+
+* <<configure-display-options,Choose how you want the element to appear on your workpad>>
+
+* <<configure-auto-refresh-interval,Change the data refresh interval>>
+
+* <<organize-element,Move and arrange elements on your workpad>>
+
+* <<element-save,Save your elements>>
+
+[float]
+[[add-canvas-element]]
+=== Add elements to your workpad
+
+Choose the elements to display on your workpad, then familiarize yourself with the element using the preconfigured demo data.
+
+. Click *Add element*.
+
+. In the *Elements* window, select the element you want to use. 
++
+[role="screenshot"]
+image::images/canvas-element-select.gif[Canvas elements]
+
+. Play around with the default settings and see what the element can do. 
+
+TIP: Want to use a different element? You can delete the element by selecting it, clicking the *Element options* icon in the top right corner, then selecting *Delete*.
+
+[float]
+[[connect-element-data]]
+=== Connect the element to your data
+
+When you are ready to move on from the demo data, connect the element to your own data.
+
+. Make sure that the element is selected, then select *Data*.
+
+. Click *Change your data source*.
+
+[float]
+[[elasticsearch-sql-data-source]]
+==== Connect to Elasticsearch SQL
+
+Access your data in Elasticsearch using the Elasticsearch SQL syntax.
+
+Unfamiliar with writing Elasticsearch SQL queries? For more information, refer to {ref}/sql-spec.html[SQL language].
+
+. Click *Elasticsearch SQL*.
+
+. In the *Elasticearch SQL query* box, enter your query, then *Preview* it.
+
+. If everything looks correct, *Save* it. 
+
+[float]
+[[elasticsearch-raw-doc-data-source]]
+==== Connect to Elasticsearch raw data
+
+Use the Lucene query syntax to use your raw data in Elasticsearch.
+
+For for more information about the Lucene query string sytax, refer to <<lucene-query,Lucene Query Syntax>>.
+
+. Click *Elasticsearch raw documents*.
+
+. From the *Index* dropdown, select the index data that you want to display. 
```

Comment:
Instead of index data, can we refer to it as "index pattern" here? The suggestions in the dropdown will only show a list of your index patterns, but you can also type in the name of an index that doesn't appear in the list (which won't show up in the dropdown if they don't match an index pattern).